src/main/kotlin/com/linker/linkerapi/notification/config/NotificationConfig.kt

```diff
@@ -0,0 +1,24 @@
+package com.linker.linkerapi.notification.config
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.linker.linkerapi.notification.api.DiscordWebhookApi
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import retrofit2.Retrofit
+import retrofit2.converter.jackson.JacksonConverterFactory
+
+@Configuration
+class NotificationConfig {
```

Comment:
```suggestion
class RetrofitConfiguration {
```

지금 보니깐 이게 더 맞을 것 같네요